### DISCUSSION TITLE
Making 1 message handler returns the same function to every input

### DISCUSSION THREAD

**Discussion by Allan1901** [python-telegram-bot/python-telegram-bot#2735] [Q&A] [answered]
I'm trying to make 1 massage handler return one same function to every input it gets. 

For example, instead of making these 2 message handler return the same function 'editing' when the input is equal to 'Edit' or 'ADD':

```
messageHandler(
                    Filters.regex('^(Edit)$'), editing),
messageHandler(
                    Filters.regex('^(ADD)$'), editing)
```

I wanted to add something like :

```
 messageHandler(
                        Filters.regex('^(%s)$' %(user_input)), editing)

```
So everything its typed returns the same function 'editing'

For a more specific case, if necessary, consider this [code](https://github.com/python-telegram-bot/python-telegram-bot/blob/master/examples/conversationbot2.py) . If, for some reason, i wanted to add only 1 message handler wich returns the function 'start' to everything the user types. How could it be done ?

**Comment by harshil21** [comment, marked as answer]
You can simply use [`Filters.text`](https://python-telegram-bot.readthedocs.io/en/stable/telegram.ext.filters.html#id6) if you want all text messages to go to one single callback.
